### ISSUE TITLE
Updating npm repo with new release anytime soon?

### ISSUE THREAD

**Issue by igornostali** [audiojs/audio#36]
Hi,

Npm repo has v1.2.0 listed https://www.npmjs.com/package/audio

Any plans to release v2.0 anytime soon?

Thanks

**Comment by dfcreative** [comment]
Hey @igornostali!
Thanks for the interest! Yes, sure, we are working hard on that — the component itself is large, it takes time to cover the listed API before 2.0. You can help project by listing API priorities you need.

**Comment by igornostali** [comment]
Thanks @dfcreative,

I was looking into joining multiple audio files into a single one. As far as I've seen this can be accomplished by :

`Audio.load(...)`

The v1.2.0 does not provide this functionality, perhaps this seems to be implemented in v2.0.0.

So, the only missing link here, I suppose, is updating npm repo,

**Comment by dfcreative** [comment]
@igornostali yep, `load(list)` and `Audio(list)` work well now with some edge-case limitations, so you can use component as github-dependency `npm i audiojs/audio`. But I'd recommend for that purpose [audio-buffer-utils concat](https://github.com/audiojs/audio-buffer-utils#utilconcatbuffer1-buffer2-buffer3-buffern-) for now as a more reliable solution.

**Comment by meszaros-lajos-gyorgy** [comment]
Hi! According to npm the last publish was 3 years ago. Any update on the new version?

**Comment by dy** [comment]
v2 is finally there
